### ISSUE TITLE
UnicodeEncodeError in sct_changelog.py ±∞

### ISSUE THREAD

**Issue by jcohenadad** [spinalcordtoolbox/spinalcordtoolbox#1782]
~~~
$SCT_DIR/python/bin/python $SCT_DIR/install/sct_changelog.py
SCT changelog -- Open milestones found 1
SCT changelog -- Latest milestone 3.2.0 21
SCT changelog -- Pull requests "3.2.0" labeled bug received 3
SCT changelog -- Pull requests "3.2.0" labeled enhancement received 3
SCT changelog -- Pull requests "3.2.0" labeled feature received 3
SCT changelog -- Pull requests "3.2.0" labeled doc received 3
SCT changelog -- Pull requests "3.2.0" labeled testing received 3
SCT changelog -- Total pull request in changelog: 12
SCT changelog -- Pull requests "3.2.0" labeled  received 3
SCT changelog -- Pull request not labelled: https://github.com/neuropoly/spinalcordtoolbox/pull/1736
SCT changelog -- Pull request not labelled: https://github.com/neuropoly/spinalcordtoolbox/pull/1746
SCT changelog -- Pull request not labelled: https://github.com/neuropoly/spinalcordtoolbox/pull/1587
SCT changelog -- Pull request not labelled: https://github.com/neuropoly/spinalcordtoolbox/pull/1735
SCT changelog -- Pull request not labelled: https://github.com/neuropoly/spinalcordtoolbox/pull/1719
SCT changelog -- Pull request not labelled: https://github.com/neuropoly/spinalcordtoolbox/pull/1781
Traceback (most recent call last):
  File "/Users/julien/code/sct/install/sct_changelog.py", line 88, in <module>
    changelog.write('\n'.join(lines))
UnicodeEncodeError: 'ascii' codec can't encode character u'\u201c' in position 972: ordinal not in range(128)
~~~

**Comment by jcohenadad** [comment]
This was caused by the use of ", #, or any greek character in the title of the PR. Until this issue is fixed, we should only stick to alphanumeric characters in PR's title.

**Comment by abtahizadeh** [comment]
Fixed via #1795 ??
Reference:
```
.encode("utf-8"))
```
